pkg/hostpath/hostpath.go

```diff
@@ -187,7 +187,7 @@ func commonUpsertRanger(obj runtime.Object, podPath, filter string, key, value i
 	// will almost always not be intermittent, but most likely the result of the
 	// host filesystem either being full or compromised in some long running fashion, so tight-loop retry, like we
 	// *could* do here as a result will typically prove fruitless.
-	// Then, the controller relist will result in going through the secrets/configmaps we share, so
+	// Then, the controller relist will result in going through the secrets/configmaps we secretShare, so
```

Comment:
```suggestion
	// Then, the controller relist will result in going through the secrets/configmaps we share, so
```

I think your search/replace got a bit overzealous here :)